contents.json

```diff
@@ -5202,6 +5202,11 @@
       "category": "authentication",
       "description": "Simple view controller to log in and retrieve an access token from LinkedIn.",
       "homepage": "https://github.com/serhii-londar/LinkedInSignIn"
+    }, {
+      "title": "swift-corelibs-xctest",
```

Comment:
Maybe it is better to change the title to "XCTest"?

Reply:
Yep, better to avoid slug as title

Reply:
fixed